sonar_dave_plugins/sonar_dave_uuv/src/gazebo_ros_image_sonar.cpp

```diff
@@ -668,68 +703,74 @@ cv::Mat GazeboRosImageSonar::ComputeNormalImage(cv::Mat& depth)
     for (int j = 0; j < normal_image.cols; ++j) {
       cv::Vec3f& n = normal_image.at<cv::Vec3f>(i, j);
-	  n = cv::normalize(n);
-   }
+      n = cv::normalize(n);
+    }
   }
 
   cv::split(normal_image.clone(), images);
   cv::Vec3d minVec, maxVec;
   for (int i = 0; i < 3; ++i) {
     cv::minMaxLoc(images[i], &minVec[i], &maxVec[i]);
-	images[i] -= minVec[i];
-	images[i] *= 1./(maxVec[i] - minVec[i]);
+    images[i] -= minVec[i];
+    images[i] *= 1./(maxVec[i] - minVec[i]);
   }
 
   cv::merge(images, normal_image);
   cv::Mat normal_image8;
   normal_image.convertTo(normal_image8, CV_8UC3, 255.0);
 
   cv_bridge::CvImage img_bridge;
-  img_bridge = cv_bridge::CvImage(this->normal_image_msg_.header, sensor_msgs::image_encodings::RGB8, normal_image8);
-  img_bridge.toImageMsg(this->normal_image_msg_); // from cv_bridge to sensor_msgs::Image
+  img_bridge = cv_bridge::CvImage(this->normal_image_msg_.header,
+                                  sensor_msgs::image_encodings::RGB8,
+                                  normal_image8);
+  // from cv_bridge to sensor_msgs::Image
+  img_bridge.toImageMsg(this->normal_image_msg_);
 
   this->normal_image_pub_.publish(this->normal_image_msg_);
 
   return normal_image;
 }
 
-cv::Mat GazeboRosImageSonar::ConstructSonarImage(cv::Mat& depth, cv::Mat& normals)
+cv::Mat GazeboRosImageSonar::ConstructSonarImage(cv::Mat& depth,
+                                                 cv::Mat& normals)
 {
   std::vector<cv::Mat> images(3);
   cv::split(normals, images);
 
-  float intensity = 100.; // target strength
-  float SL = 200.; // source level
-  float NL = 30; // noise level
-  float DI = 0.0; // directivity index
+  float intensity = 100.;  // target strength
+  float SL = 200.;         // source level
+  float NL = 30;           // noise level
+  float DI = 0.0;          // directivity index
 
   if (dist_matrix_.empty()) {
     std::vector<float> t_x, t_y;
-    for (int i = 0; i < depth.cols; i++) t_x.push_back((float(i) - this->cx_)/this->focal_length_);
-    for (int i = 0; i < depth.rows; i++) t_y.push_back((float(i) - this->cy_)/this->focal_length_);
+    for (int i = 0; i < depth.cols; i++)
+      t_x.push_back((static_cast<float>(i) - this->cx_)/this->focal_length_);
+    for (int i = 0; i < depth.rows; i++)
+      t_y.push_back((static_cast<float>(i) - this->cy_)/this->focal_length_);
     cv::Mat X, Y;
-    cv::repeat(cv::Mat(t_x).reshape(1,1), t_y.size(), 1, X);
-    cv::repeat(cv::Mat(t_y).reshape(1,1).t(), 1, t_x.size(), Y);
+    cv::repeat(cv::Mat(t_x).reshape(1, 1), t_y.size(), 1, X);
+    cv::repeat(cv::Mat(t_y).reshape(1, 1).t(), 1, t_x.size(), Y);
     dist_matrix_ = cv::Mat::zeros(depth.rows, depth.cols, CV_32FC1);
     cv::multiply(X, X, X);
     cv::multiply(Y, Y, Y);
     cv::sqrt(X + Y + 1, dist_matrix_);
   }
 
   // TODO: make these into proper parameters
-  cv::Mat TS = intensity*images[2]; // target strength, probably dir should be DI
```

Comment:
Recommend not removing comments that the writer put in.  It likely provides meaning and was placed intentionally.  Maybe move whole comment to line above.

Reply:
This file will be removed when issue 25 is addressed.

Reply:
Bruce, was this your comment? If so, could you clarify what it meant? There were many comments in the code that appeared to be "notes to self" and it seemed the file was in a draft state.

Reply:
No, all these comments were inherited.

But please disregard all this; I'll delete this file today as part of fixing issue 25.

Reply:
I see. OK.